src/services/cherry-picker.ts

```diff
@@ -156,6 +161,26 @@ export class CherryPicker {
     return node
   }
 
+  // Fetch app/node's service log from redis
+  async fetchRawServiceLogs(blockchain: string, rawNodeIDs: string[]): Promise<{ [id: string]: string }> {
+    const redisKeys = [] as string[]
```

Comment:
```suggestion
    const redisKeys : string[] =  []
```

Reply:
`as` is better when you're casting from one type to another, on this case as you're defining the type is better to simply put it at the definition

Reply:
line was removed